nrpy/infrastructures/BHaH/general_relativity/ADM_Initial_Data_Reader__BSSN_Converter.py

```diff
@@ -341,10 +341,7 @@ def Cfunction_BSSN_Cart_to_rescaled_BSSN_rfm(
         body += r"""
   REAL xx0,xx1,xx2 __attribute__((unused));  // xx2 might be unused in the case of axisymmetric initial data.
   {
-    int unused_Cart_to_i0i1i2[3];
-    REAL xx[3];
-    Cart_to_xx_and_nearest_i0i1i2(commondata, params, xCart, xx, unused_Cart_to_i0i1i2);
-    xx0=xx[0];  xx1=xx[1];  xx2=xx[2];
+    xx0=xxL[0];  xx1=xxL[1];  xx2=xxL[2];
```

Comment:
Please just set this above in line 342. Also please remove the __attribute__((unused)). Thanks!